src/main/cpp/t05_ave.cpp

```diff
@@ -18,9 +18,21 @@
 #include "t05_ave.h"
 #include <iostream>
 #include <iomanip>
+#include <cmath>
 
 using namespace std;
 
 int t05_ave() {
-
+double s = 0.0, n;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'n' is not assigned a value.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4430873)